packages/node/src/backend.ts

```diff
@@ -19,6 +19,9 @@ import { HTTPSTransport, HTTPTransport } from './transports';
  * @see NodeClient for more information.
  */
 export interface NodeOptions extends Options {
+  /** handler */
+  aws_context?: string;
```

Comment:
`aws_context` cannot live on the `Options`, because it's changing for each invocation. Is should be provided inside the integration handler directly.